src/OpenTelemetry.Instrumentation.EventCounters/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## 1.0.0-alpha.3
```

Comment:
How about updating this to `1.5.0-alpha.1`?

Reply:
because this release contains bug fixes?

Reply:
No because we are doing this for the main repo instrumentation packages and also Runtime instrumentation package on this repo. You can find the reason for doing that here: https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Instrumentation.AspNetCore/CHANGELOG.md#150-beta1